### PR TITLE
[Ronin] Add ronin prefix to address

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes https://github.com/trustwallet/wallet-core/issues/1980

### CODE REVIEW

**Review by catenocrypt — APPROVED** [review]

File: src/Ethereum/Entry.cpp [hunk 1]
```diff
@@ -7,22 +7,38 @@
...
 
 #include "Address.h"
+#include "Ronin/Address.h"
 #include "Signer.h"
 
 using namespace TW::Ethereum;
 using namespace std;
 
 bool Entry::validateAddress(TWCoinType coin, const string& address, TW::byte, TW::byte, const char*) const {
-    return Address::isValid(address);
```
Comment (vikmeup) [inline src/Ethereum/Entry.cpp:16]: I would prefer to keep ronin type addresses separate from ethereum to avoid any future bugs or confusion

Comment (hewigovens) [reply src/Ethereum/Entry.cpp:16]: You meant add another entry point? Ronin is exactly ethereum with ronin: prefix, will still need to call ethereum code

Comment (hewigovens) [reply src/Ethereum/Entry.cpp]: done

Comment (farisei77) [reply src/Ethereum/Entry.cpp]: Rather most thinkeble

**Review by w20089527 — APPROVED** [review]

**Pull request merged**